internal/app/api/v1/executions.go

```diff
@@ -509,6 +501,55 @@ func (s TestkubeAPI) GetExecuteOptions(namespace, id string, request testkube.Ex
 	}, nil
 }
 
+// streamLogsFromStorage writes logs from the output of executionResult to the writer
```

Comment:
Why from storage? is there any database used here? As name suggests that it's fetched from some "storage"

Reply:
Storage as opposed to K8s job logs. It makes more sense in the method that it is called from, but there is not direct connection to the storage in this one, only the ExecutionResult that's already queried from there.

would streamFinishedExecutionLogs() be a better name for this?

Reply:
It streams execution result if I understand correctly what it does?

Reply:
Yes, it streams the result from the ResultRepository

Reply:
this function just streams ExecutionResult (not from repository as it's not fetching it) executionResult is passed from outside wherever it'll be so context of this function is not related to repository at all.

Reply:
fair point, updated it to `streamLogsFromResult(...)`